### ISSUE TITLE
env: illegal option -- 0 error

### ISSUE THREAD

**Issue by christian-bromann** [stateful/vscode-runme#302]
Running any code cell seems to fail due to this weird error:

```
 *  Executing task: exec: echo "hoi" (RUNME_ID: /Users/christianbromann/Sites/Stateful/projects/gosh-cli/README.md:11) 

env: illegal option -- 0
usage: env [-iv] [-P utilpath] [-S string] [-u name]
           [name=value ...] [utility [argument ...]]
hoi
env: illegal option -- 0
usage: env [-iv] [-P utilpath] [-S string] [-u name]
           [name=value ...] [utility [argument ...]]

 *  The terminal process failed to launch (exit code: 1). 
 *  Terminal will be reused by tasks, press any key to close it. 
```

I am not quite sure where `env` is being executed, it doesn't seem to be related to what is executed.

**Comment by christian-bromann** [comment]
This seems to be unrelated to the VS Code extension as I get the same when just using the CLI:

```
❯ which runme
/opt/homebrew/bin/runme
❯ runme --version
runme version 0.6.5 (e6063de0f74b70e8312e8d8f14c675ad4117c054) on 2023-03-09T15:49:50Z
❯ runme run install
env: illegal option -- 0
usage: env [-iv] [-P utilpath] [-S string] [-u name]
           [name=value ...] [utility [argument ...]]
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

changed 107 packages, and audited 108 packages in 5s

6 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
tonos_cli 0.32.0
COMMIT_ID:
BUILD_DATE: 2023-03-07 10:00:13 -0800
COMMIT_DATE:
GIT_BRANCH:
env: illegal option -- 0
usage: env [-iv] [-P utilpath] [-S string] [-u name]
           [name=value ...] [utility [argument ...]]
failed to run command "install": exit code: 1
```

**Comment by christian-bromann** [comment]
The problem for this bug is that Mac users have a BSD version of `env` which doesn't support `-0` as a flag. To workaround this you have to install GNUs version of it:

```
❯ brew install coreutils
# find GNUs env binary
❯ find /opt/homebrew /usr/local -name genv -print
/opt/homebrew/bin/genv
/opt/homebrew/Cellar/coreutils/9.1/bin/genv
# create a copy named `env` so it takes precedence
❯ cp /opt/homebrew/bin/genv /opt/homebrew/bin/env
❯ which env
/opt/homebrew/bin/env
```

**Comment by mxsdev** [comment]
Closing as fixed in stateful/runme#190
